drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -104,7 +104,7 @@ def getTimeStamp():
 
 def copyLogFilesTo(logFiles, filePath, targetFolderPath):
     for logFile in logFiles:
-        src = os.path.join(filePath, logFile)
+        src = os.path.join(filePath, logFile.getName())
```

Comment:
what is `src`? The source path? According to Clean Code best practise, we shall use declarative variable names.

Reply:
changed